tests/conftest.py

```diff
@@ -71,3 +74,32 @@ def pathological_table(client):
     db.session.add(csv_file)
     db.session.commit()
     yield csv_file
+
+
+def _find_free_port():
+    with contextlib.closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
+        sock.bind(('', 0))
+        return sock.getsockname()[1]
+
+
+@pytest.fixture
+def opencpu_request(app):
+    import docker
+    from time import sleep
+    from viime.opencpu import opencpu_request
+
+    client = docker.from_env()
+    client.images.build(path=dockerfile, tag='viime')
+    port = _find_free_port()
+    container = client.containers.create('viime', ports={
+        '8004/tcp': port})
+
+    app.config['OPENCPU_API_ROOT'] = 'http://localhost:%s/ocpu/library' % port
+
+    container.start()
+    sleep(5)
+
+    yield opencpu_request
+
+    container.stop()
+    container.remove()
```

Comment:
I would rather avoid mixing build-time concerns with testing.  The tests should just assume the container is running at the URI configured.  If we want to add starting up the opencpu service with the test environment [tox-docker](https://github.com/tox-dev/tox-docker) is a viable option.